### PR TITLE
20.12.0-rns

### PR DESCRIPTION
**Pull request by moishce**
## Status
In Progress

### CODE REVIEW

File: content-repo/extra-docs/releases/20.12.0.md [hunk 1]
```diff
@@ -0,0 +1,784 @@
...
+Several packs include breaking changes.
+
+- [Check Point Firewall Pack v2.0.5](#check-point-firewall-pack-v205)
+- [CrowdStrike Falcon Sandbox Pack v1.0.3](#crowdstrike-falcon-sandbox-pack-v103)
+
+### New: Google Drive Pack v1.0.0
+#### Classifiers
+##### Google Drive - Classifier
+Classifies Google Drive Incidents.
+##### Google Drive
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.12.0.md:14]:
```suggestion
```

File: content-repo/extra-docs/releases/20.12.0.md [hunk 2]
```diff
@@ -0,0 +1,784 @@
...
+The resource name of the shared drive.
+- **Google Drive Activity Copied Shared Drive Title**
+The title of the shared drive.
+- **Google Drive Activity Created New**
+Indicates the object was newly created (e.g. as a blank document), not derived from a Drive object or external object.
+- **Google Drive Activity DLP Change Type**
+The type of Data Leak Prevention (DLP) change.
+- **Google Drive Activity Delete Type**
+The type of delete action taken.
+- **Google Drive Activity Identified As**
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.12.0.md:51]: add description here

File: content-repo/extra-docs/releases/20.12.0.md [hunk 3]
```diff
@@ -0,0 +1,784 @@
...
+The removed parent object(s).
+- **Google Drive Activity Removed Permissions**
+The set of permissions removed by this change.
+- **Google Drive Activity Restore Type**
+The type of restore action taken.
+- **Google Drive Activity Restriction Changes**
+The set of changes made to restrictions.
+- **Google Drive Activity Suggestion Subtype**
+The sub-type of this event.
+- **Google Drive Activity Targets**
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.12.0.md:73]: add description here

File: content-repo/extra-docs/releases/20.12.0.md [hunk 4]
```diff
@@ -0,0 +1,784 @@
...
+
+### New: Microsoft Graph API Pack v1.0.0
+#### Integrations
+##### Microsoft Graph API
+Use the Microsoft Graph API integration to interact with Microsoft APIs that do not have dedicated integrations in Cortex XSOAR, for example, Mail Single-User, etc.
+
+
+---
+
+### New: RSS Pack v1.0.0
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.12.0.md:110]:
```suggestion
### New: RSS Pack v1.0.0 (Community Contributed)
```

File: content-repo/extra-docs/releases/20.12.0.md [hunk 5]
```diff
@@ -0,0 +1,784 @@
...
+- **Affected Entities**
+
+- **Affected Entities List**
+
+- **Average Complexity Level**
+
+- **Average Complexity of Compromise**
+
+- **Compromising Techniques**
+Name of each attack technique that has compromised the entity and the count such attack vectors from the time period
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.12.0.md:138]: either remove incident fields description or fill for them all

File: content-repo/extra-docs/releases/20.12.0.md [hunk 6]
```diff
@@ -0,0 +1,784 @@
...
+---
+
+### AlphaSOC Network Behavior Analytics Pack v1.0.2 (Partner Supported)
+#### Integrations
+##### AlphaSOC Network Behavior Analytics
+Fixed an issue where the minimum severity was not used when fetching incidents.
+
+---
+
+### ApiModules Pack v2.1.0
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.12.0.md:250]: remove api modules pack

File: content-repo/extra-docs/releases/20.12.0.md [hunk 7]
```diff
@@ -0,0 +1,784 @@
...
+- Documentation and metadata improvements.
+- Added **Panorama.Monitor.Logs.Vsys** output field in the ***panorama-get-logs*** command.
+
+#### Playbooks
+##### Panorama Query Logs
+Added **Panorama.Monitor.Logs.Vsys** in the sub-playbook outputs.
+
+---
+
+### PANW Comprehensive Investigation Pack v1.3.4
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.12.0.md:666]: remove this

File: content-repo/extra-docs/releases/20.12.0.md [hunk 8]
```diff
@@ -0,0 +1,784 @@
...
+
+### VMware Pack v1.1.0
+#### Integrations
+##### VMware
+- General documentation improvements.
+- Added the ***vmware-change-nic-state*** command which changes the state of a VM NIC.
+
+---
+
+### XM Cyber Pack v1.0.1 (Partner Supported)
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.12.0.md:757]: please unify this with the new pack

File: content-repo/extra-docs/releases/20.12.0.md [hunk 9]
```diff
@@ -0,0 +1,784 @@
...
+### New: XM Cyber Pack v1.0.0 (Partner Supported)
+#### Classifiers
+##### XM Cyber Incident Classifier
+Classifies XM Cyber alerts and events into 4 types.
+##### XM Cyber Entity from Crowdstrike
+Maps Crowdstrike Falcon incident data to XM Cyber fields for automatic enrichment.
+##### XM Cyber Incident Mapper
+Maps fields from XM Cyber incidents.
+#### Incident Fields
+- **# Critical Assets at Risk**
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.12.0.md:127]:
```suggestion
- **Critical Assets at Risk**
```

**Comment by kirbles19** [comment]
@moishce Done, ready to go.

**Review by Itay4 — APPROVED** [review]

**Pull request merged**